.travis.yml

```diff
@@ -13,9 +13,15 @@ addons:
 before_install:
   - cat test-requirements.txt >> requirements.txt
 
+before_script:
+  - rm -f static/errors
```

Comment:
wrt https://github.com/manu-chroma/username-availability-checker/pull/96/commits/3f4a959517843316a13e5939e6825b4df8a2b7dc , I dont believe Travis is caching test results.
Can you explain how that is happening?

Reply:
hmm... I thought that the dump was increasing... but it seems it's not.
I'll rebase the commit